### PR TITLE
Add gitlab runner to Software exposing Prometheus metrics

### PR DESCRIPTION
**Pull request by oussemos**
Add gitlab runner in Software exposing Prometheus metrics

### CODE REVIEW

**Comment by brian-brazil** [comment]
https://gitlab.com/gitlab-org/gitlab-runner/blob/master/executors/docker/machine/provider.go#L431

Most of this seems to be custom collectors, this is the only direct instrumentation I can find and it is procedurally generating metric names which is a no-no. That should be a label.

My idea is that directly instrumented code listed here provides prime examples of what Prometheus instrumentation is meant to look like.

File: content/docs/instrumenting/exporters.md [hunk 1]
```diff
@@ -189,6 +189,7 @@ separate exporters are needed:
    * [SkyDNS](https://github.com/skynetservices/skydns) (**direct**)
    * [Telegraf](https://github.com/influxdata/telegraf/tree/master/plugins/outputs/prometheus_client)
    * [Weave Flux](https://github.com/weaveworks/flux)
+   * [Gitlab runner](https://docs.gitlab.com/runner/monitoring/README.html) (**direct**)
```
Comment (brian-brazil) [inline content/docs/instrumenting/exporters.md]: Keep it sorted.

Comment (oussemos) [reply content/docs/instrumenting/exporters.md]: Done

**Comment by tmaczukin** [comment]
@brian-brazil Thanks for noticing this. We're currently in the process of cleaning-up the metrics naming in GitLab Runner, see https://gitlab.com/gitlab-org/gitlab-runner/issues/3114.

**Pull request closed without merging**